tests/addresslib/parser_address_list_test.py

```diff
@@ -16,14 +15,8 @@ def powerset(iterable):
     return chain.from_iterable(combinations(s, r) for r in range(len(s)+1))
 
 @nottest
-def run_relaxed_test(string, expected_mlist, expected_unpar):
```

Comment:
Most of the tests removed or updated in this file are due to the fact that we removed the distinction between `relaxed` and `strict` processing of lists. See the PR description for more information.